pages.zh/linux/lvm.md

```diff
@@ -0,0 +1,27 @@
+# lvs
+
+> 显示逻辑卷信息。
+> 另见: `lvm`。
```

Comment:
```suggestion
> 另见: `lvm`.
```

See rule 5 in [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.zh.md#%E4%B8%AD%E8%A5%BF%E6%96%87%E6%B7%B7%E6%8E%92%E8%A7%84%E5%88%99).